### PR TITLE
made shooter command to search for target

### PR DESCRIPTION
**Pull request by skyler-ferrante**

### CODE REVIEW

**Review by DannyGramowski — APPROVED** [review]
Works good

Was made by Edward on Danny's computer. My bad

File: src/main/java/frc/shooter/command/ScanForTarget.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import frc.drives.DrivesSensorInterface;
+import frc.drives.DrivesSensors;
+import frc.sensors.Limelight;
+import frc.shooter.ShooterCommand;
+import frc.shooter.ShooterOutput;
+import frc.shooter.ShooterSensorsInterfeace;
+
+public class ScanForTarget extends ShooterCommand {
+
+    Limelight l;
```
Comment (secretspy97) [inline src/main/java/frc/shooter/command/ScanForTarget.java]: Unused

File: src/main/java/frc/shooter/command/ScanForTarget.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    final int maxAngleOnEitherSide = 160;
+    final int howCloseToEdge = 3;
+
+    public ScanForTarget(ShooterSensorsInterfeace sensors, DrivesSensorInterface driveSensors){
+        super(sensors , driveSensors);
+        l = new Limelight();
```
Comment (secretspy97) [inline src/main/java/frc/shooter/command/ScanForTarget.java]: unused

File: src/main/java/frc/shooter/command/ScanForTarget.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                return new ShooterOutput(.2);
+            }
+            return new ShooterOutput(-.2);
+        }else{
+            if(angle>-maxAngleOnEitherSide+howCloseToEdge){
+                return new ShooterOutput(-.2);
+            }
+            return new ShooterOutput(.2);
+        }
+        System.out.println("ScanForTarget.java, logic error");
```
Comment (secretspy97) [inline src/main/java/frc/shooter/command/ScanForTarget.java]: Unreachable statement - and below

File: src/main/java/frc/shooter/command/ScanForTarget.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        l = new Limelight();
+    }
+
+    @Override 
+    public ShooterOutput execute() {
+        double angle = sensors.getShooterAngleToRobot();
+
+        //On right side
+        if(angle > 0){
+            if(angle<maxAngleOnEitherSide-howCloseToEdge){
```
Comment (secretspy97) [inline src/main/java/frc/shooter/command/ScanForTarget.java]: This will go to the edge, flip direction and then isolate on the edge, not scanning

**Review by secretspy97 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**